client/src/pages/DriverLocationInfo/DriverLocationInfo.tsx

```diff
@@ -62,18 +84,26 @@ function processDriversForTable(
         phoneNumber={curDriver.phoneNumber}
       />,
     ];
+
   });
+  
 }
 
+
+
 //TODO: Make this code cleaner
 function processDropoffLocationsForTable(
   drivers: ProcessedDriver[],
   processedDropOffLocations: ProcessedDropoffLocation[]
 ) {
   return processedDropOffLocations.map((curLocation, i) => {
+    const matchedDrivers = drivers.filter((d) =>
+      d.dropoffLocations.includes(curLocation.id)
+    );
+
```

Comment:
What is this code doing? Doesn't seem related to the overall PR for just adding the "No data available"

Reply:
Oh, I see, it's for below to have count of drivers for the message.